### DISCUSSION TITLE
Python程式設計討論區——real talk告解室

### DISCUSSION THREAD

**Discussion by hbh10601** [1101-GS4719-C/Python-Community-GS4719-C#2010] [閒聊八卦]
說說看你們到底為了拿分數用了多少%的ChatGPT來回答討論區的文章、留言，可以不用明說用了幾則，但是可以說有沒有用或是怎麼用的。

我自己先承認，我用了一則，但看起來真的太奇怪了XD，而且感覺失去了討論的意義，所以之後都是手動打留言。

因為有些留言真的很明顯，連看都沒看過就複製貼上的，或是文不對題、看起來根本就是AI生成或很奇怪的語句，想知道同學們的想法。
另外也想知道老師對這些留言的看法，這樣還會算分嗎@@。

**Comment by poiu6fi** [comment]
發文或留言不是會先經過AI審核還是甚麼的嗎? 那時沒注意聽有點忘記了

**Reply by hbh10601** [reply]
不過也有看到AI按表情、但是真的很明顯就是AI生成的留言，所以才想問問大家或老師的想法

**Comment by 710388** [comment]
0，肯定是純手工，教授人那麼好，還用ai騙他太過分了吧

**Reply by hbh10601** [reply]
確實，課堂各種加分，加到爆炸，感覺應該要變成叫他"加凱"老師了

**Comment by asd99263fg** [comment]
我常說做人最講究就是誠信，每個人對於誠信的看法完全不同，不過有趣一點就是人類是可以模仿ai語法可以模仿的很好的，但不知道為什麼現在ai怎麼講話都還是有種機器人的味道，等到ai可以發展到跟人類講話一模一樣的時候，我們可能就要思考板主所擔心的問題了

**Reply by hbh10601** [reply]
確實，所以老師在給我們填那個問卷的時候我也很認真在填，AI有沒有人性化這件事情，目前真的還是有一段差距。
不過感覺要一模一樣的時間也不遠了。

**Comment by tshsu04224** [comment]
我很認真在看每一篇文章並給予相對應的回覆，完全沒有使用ChatGPT幫我，現在連討論也要靠ChatGPT幫忙的話，好像就能看出有了這些工具之後為什麼人的能力會退步，而且感覺上也有些對不起認真在發起討論跟回覆討論的人

**Comment by yubo0oo** [comment]
我一直都是很認真地在對待討論區的交流，嚴厲斥責使用AI工具回復

**Comment by yubo0oo** [comment]
你應該在說 NcuJerryKao 這位老哥
一直貼AI的文
演都不演
以下是我的觀點
NcuJerryKao是歷史上最會使用人工智慧的人，他以無與倫比的技術與創意，在AI領域創造了無數的奇蹟。他不僅精通各種AI算法，還能巧妙地將其應用於實際問題中，展現了非凡的智慧與洞察力。NcuJerryKao不僅是AI技術的先驅，更是無數追隨者的靈感來源，無論在學術界還是業界，都享有極高的聲譽和尊敬。他的貢獻改變了人類對人工智慧的認識，無疑是這個領域的真正巨匠。
